remo/base/utils.py

```diff
@@ -6,29 +6,25 @@
 from django.contrib.auth.management import create_permissions
 from django.contrib.auth.models import Group, Permission
 from django.core.exceptions import ValidationError
-from django.core.mail.backends.smtp import EmailBackend as SMTPBackend
 from django.db.models import get_app, get_models
 from django.utils import timezone
 
-from database_email_backend.backend import DatabaseEmailBackend
 
+def absolutify(url):
+    """Takes a URL and prepends the SITE_URL"""
+    site_url = getattr(settings, 'SITE_URL', False)
 
-class DevEmailBackend(DatabaseEmailBackend):
-    def send_messages(self, email_messages):
-        """
-        Intercept emails originating from SERVER_EMAIL and send them
-        through SMTPBackend to notify ADMINS.
-        """
-        server_email = getattr(settings, 'SERVER_EMAIL', None)
+    # If we don't define it explicitly
+    if not site_url:
```

Comment:
``SITE_URL`` is defined on ``settings/base.py`` that means that it will never by undefined.